docs/website/docs/general-usage/credentials/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Configuration and Secrets
+description: How to configure dlt pipelines and set up credentials
+keywords: [credentials, secrets.toml, secrets, config, configuration, environment variables]
+---
+import DocCardList from '@theme/DocCardList';
+
+`dlt` pipelines usually require configurations and credentials. These can be set up in [various ways](setup):
```

Comment:
```suggestion
`dlt` pipelines usually require configurations and credentials (secrets):
- **Configuration** refers to non-sensitive settings that define a data pipeline's behavior. These include file paths, database hosts, timeouts, API URLs, and performance settings.
- **Secrets** are sensitive data like passwords, API keys, and private keys. They should never be hard-coded to avoid security risks. 
These can be set up in [various ways](setup):
```

I would add a short definition of configuration and secrets:

Reply:
Hm I wanted to keep this section short otherwise the other sections wouldn't be visible
<img width="902" alt="image" src="https://github.com/user-attachments/assets/45603663-170b-41f8-be5a-06bb44daaea1">

Reply:
I don't think it's a big problem. The "Details" will still be visible.

Reply:
Added this info to `How to set up credentials` because I really want the index page to be short